src/GlobalsAttributeReader.php

```diff
@@ -0,0 +1,119 @@
+<?php
+declare(strict_types=1);
+
+namespace Zalas\PHPUnit\Globals;
+
+use PHPUnit\Event\Code\TestMethod;
+use PHPUnit\Event\Test\Prepared;
+use PHPUnit\Event\Test\PreparedSubscriber;
+use Zalas\PHPUnit\Globals\Attribute\Env;
+use Zalas\PHPUnit\Globals\Attribute\Putenv;
+use Zalas\PHPUnit\Globals\Attribute\Server;
+
+final class GlobalsAttributeReader implements PreparedSubscriber
```

Comment:
Looks like we need to use `PreparationStartedSubscriber` instead of `PreparedSubscriber` to be consistent with the behaviour for annotations before PHPUnit 10.

See the recent bug fix: https://github.com/jakzal/phpunit-globals/pull/35

Reply:
Ah nevermind. I see. I will rebase with latest main and adjust this, so we don't introduce the same bug again 👍

Reply:
Changed it to `PreparationStartedSubscriber`